src/Text/Pretty/Simple.hs

```diff
@@ -725,6 +754,20 @@ layoutStringAnsi opts = fmap convertStyle . layoutString opts
 -- >>> pPrintString "0x1b"
 -- 0x1b
 --
+-- >>> pPrintString "[div_ [class_ \"level\"] [button_ [onClick False] [text \"-\"], div_ [] [text $ ms level], button_ [onClick True] [text \"+\"]]]"
```

Comment:
This is pretty far outside of the realm of what `pretty-simple` is designed to support, but it is pretty awesome that it works so well. And this was the motivating input for this change, although I think it's positive more generally.